treeherder/etl/buildapi.py

```diff
@@ -174,6 +174,16 @@ def transform(self, data, filter_to_project=None, filter_to_revision=None,
             else:
                 log_reference = []
 
+            # add structured logs to the list of log references
+            if 'blobber_files' in prop:
+                blobber_files = json.loads(prop['blobber_files'])
+                for bf, url in blobber_files.items():
+                    if bf.endswith('_raw.log'):
+                        log_reference.append({
+                            'url': url,
+                            'name': 'structured-raw'
```

Comment:
For the text logs, we've used "text_log_summary" for the artifact name, and above I proposed using "text_log" or "buildbot_text_log" for the name in job_log_urls. Whereas the naming here will be "structured_log_raw" and "structured_log_summary". I'd like us to be consistent and either use "raw" for both, or for neither. Also, "structured" could apply to any structured log type, and we may use something other than json in the future. How about "json_log" or "blobber_json_log" or ...?